recipes/phabox/meta.yaml

```
@@ -1,61 +1,64 @@
-    - pandas >=2
-    - prodigal-gv >=2.11.0
-    - pyarrow >=16
-    - pytorch >=2.4
-    - scipy >=1.14
-    - seaborn-base >=0.13.2
-    - transformers >=4
+    - python =3.10.14
+    - diamond =0.9.14
+    - blast =2.16.0 
+    - mcl =22.282
+    - fasttree =2.1.11 
```

yamllint

[error] trailing spaces

(trailing-spaces)